extensions/linz/README.md

```diff
@@ -61,18 +61,18 @@ See [ISO/IEC 13249-3:2016(en)](https://www.iso.org/obp/ui/#!iso:std:60343:en) fo
+| summaries/created/minimum      | date-time            | **REQUIRED**. Earliest [asset created value](#asset-fields), in UTC.                                                                                                                                                                                                                                                                             |
+| summaries/created/maximum      | date-time            | **REQUIRED**. Latest [asset created value](#asset-fields), in UTC.                                                                                                                                                                                                                                                                               |
+| summaries/updated/minimum      | date-time            | **REQUIRED**. Earliest [asset updated value](#asset-fields), in UTC.                                                                                                                                                                                                                                                                             |
+| summaries/updated/maximum      | date-time            | **REQUIRED**. Latest [asset updated value](#asset-fields), in UTC.                                                                                                                                                                                                                                                                               |
+| summaries/linz:geospatial_type | \[string]            | **REQUIRED**. A general description of the type of content that can be found in the dataset. See the [list of accepted geospatial types](#geospatial-types).                                                                                                                                                                                     |
+| linz:lifecycle                 | string               | **REQUIRED**. Lifecycle Status of Collection. Must be one of `under development`, `preview`, `ongoing`, `completed`, `deprecated`.                                                                                                                                                                                                               |
+| linz:providers                 | LINZ Provider Object | **REQUIRED**. The object provides information about a provider. A provider is any of the organizations that captures or processes the content of the assets and therefore influences the data offered by the STAC implementation. See [LINZ Provider Object](#linz-provider-object).                                                             |
```

Comment:
I think might want to say why we have a `linz:providers` field in addition to `providers`. 
Something like `LINZ specific provider 'roles'` at the start of the description?